clientv3/client.go

```diff
@@ -215,6 +216,11 @@ func (c *Client) dialSetupOpts(endpoint string, dopts ...grpc.DialOption) (opts
 	if c.cfg.DialTimeout > 0 {
 		opts = []grpc.DialOption{grpc.WithTimeout(c.cfg.DialTimeout)}
 	}
+	if c.cfg.DialKeepAlive > 0 {
+		opts = append(opts, grpc.WithKeepaliveParams(keepalive.ClientParameters{
+			Time: c.cfg.DialKeepAlive * time.Second,
```

Comment:
We want to read the input in seconds, but send it as time.Duration. time.Duration can be anything (it is nanoseconds in this case). Maybe I should change the type of `c.cfg.DialKeepAliveTime` to int to make it clear?